libs/blocks/reading-time/reading-time.js

```diff
@@ -28,5 +30,19 @@ export default async function init(el) {
   const format = await replaceKey(placeholderKey, config);
   const readingTimeText = format.replace(/[1,5]/g, readingTime);
   span.innerHTML = readingTimeText;
-  el.append(span);
+  block.append(span);
+  if (block.classList.contains('inline')) {
+    const section = block.closest('.section');
+    const inlineSiblinggs = section.querySelectorAll('.inline');
+    if (inlineSiblinggs.length > 1) {
+      let inlineContainer = section.querySelector('.inline-wrapper');
+      if (!inlineContainer) {
+        inlineContainer = createTag('div', { class: 'inline-wrapper content' });
+        block.after(inlineContainer);
+      }
+      inlineSiblinggs.forEach((el) => inlineContainer.append(el));
+    } else {
+      block.classList.add('content');
+    }
```

Comment:
This whole block of code is the exact same as [what's in share.js](https://github.com/adobecom/milo/pull/467/files#diff-71d2d1ba42c460262b54dc2d8c6a6195d8112dd9b65e509dcbb19d979eb22d83R122-R133).
So, the exact code runs multiple times every time there are sibling inline variants. 

We need a better way to handle this. Inline-utility.js or something.